gapic/src/main/com/google/gapid/perfetto/views/GpuQueuePanel.java

```diff
@@ -204,6 +206,7 @@ protected Hover onTrackMouseMove(Fonts.TextMeasurer m, double x, double y, int m
           hoveredCategory = "";
         }
         hoveredTitle = buildSliceTitle(hoveredTitle, data.args[i]);
+        hoveredTitle += " (" + TimeSpan.timeToShortString(tEnd -tStart) + ")";
```

Comment:
Use `timeToString` instead of the short version. Precision here matters and slices are unlikely to be too long to warrant the short version. (ditto below)

Reply:
done